lib/iex/test/iex/helpers_test.exs

```diff
@@ -382,6 +382,15 @@ defmodule IEx.HelpersTest do
       assert capture_io(fn -> h(def) end) =~ def_h
     end
 
+    test "indents after line break in long specs" do
+      spec =
+        "\n" <>
+          "  @spec ascii_printable?([...], limit) :: boolean()\n" <>
+          "          when limit: :infinity | pos_integer()\n"
```

Comment:
Also note that Elixir doesn't indent `when`. This is how Elixir formats this code:

```elixir
@spec ascii_printable?([...], limit) :: boolean()
      when limit: :infinity | pos_integer()
```

Reply:
If we consider Elixir's indention, the code is actually correct. So I will close this. Thank you!